.github/workflows/build-and-deploy-notifications.yml

```diff
@@ -64,6 +64,12 @@ jobs:
           report_name: "Test results"
           dotnet_test_args: '-e ConnectionStrings__DefaultConnection="Data Source=(local); Initial Catalog=${{ env.SQL_DB }}; User=sa; Password=${{ env.SQL_PASSWORD }}; TrustServerCertificate=True"'
 
+      - name: Upload artifact for deployment job
```

Comment:
No steps have been added to consume this artifact so it might be better off in a different PR.  It will also probably need to be more selective in what's uploaded for deployment to the function app service

Reply:
Okay make sense

Reply:
removed